src/main/java/com/exasol/adapter/document/edml/deserializer/MappingDeserializer.java

```diff
@@ -13,23 +13,23 @@
 
 class MappingDeserializer {
     static void deserializeToColumnMapping(final JsonObject json,
-            final AbstractToColumnMapping.AbstractToColumnMappingBuilder<?, ?> builder) {
+                                           final AbstractToColumnMapping.AbstractToColumnMappingBuilder<?, ?> builder) {
```

Comment:
Could it be that your formatter settings differ from default?

Reply:
It was. Good you noticed. Fixed this now.